docs/sources/flow/when-to-use-flow/opentelemetry-collector.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Learn how to Flow compares to the OpenTelemetry Collector
+menuTitle: OpenTelemetry Collector
+title: OpenTelemetry Collector
+weight: 100
+---
+
+# Comparing Flow with the OpenTelemetry Collector
+
+## Which one of the two should you use? 
+
+Flow may be a better choice if:
+* You prefer River syntax over YAML.
+* You would like to collect "profile" telemetry - Flow can do this via Pyroscope components.
+* You would like to process Prometheus or Loki telemetry without converting it to OTLP.
+
+The Collector may be a better choice if:
+* You are already using the Collector, and do not need any of the features of Flow.
+* You prefer YAML syntax over River.
+
+Below you can find more details on similarities and differences between the Collector and Flow.
+
+## Similarities
+
+### Many Collector components are available in Flow
+
+Flow contains many of the components available inside the Collector:
+* For example, the OTLP exporter is available as `otelcol.exporter.otlp`.
+* The Grafana Agent development team is actively working on adding more of the Collector's components to Flow.
+* If a Flow user needs a Collector feature to be made available in the Agent, the Agent development team could work on implementing it.
+
+### Similar performance when processing OpenTelemetry signals natively
+
+Most of Flow's `otelcol` components are just thin wrappers over a Collector component.
+Hence, the CPU and memory performance of Flow is similar.
+
+## Differences
+
+### Configuration language
+
+Collector is configured using yaml, whereas Flow is configured using River.
+
+#### Example - coalesce
+
+One of River's main advantages is its standard library. It contains handy functions such as coalesce, 
+which could be used to retrieve the first argument which is not null or empty:
+
+```river
+```
+
+### Modularity
+
+The Agent configuration is more flexible, modular, and allows for more opportunities to chain components together in a pipeline. 
```

Comment:
```suggestion
The Agent configuration is more flexible and modular, allowing you to chain more components together in a pipeline. 
```

I'm not sure if I'm changing the meaning of the sentence here.  Do you want  to focus on "more opportunities" to chain different sets of components into a single pipeline... or the ability to link together more components as a single set?

Reply:
That adds a LOT more information. :-)